android/sdl_android/src/main/java/com/smartdevicelink/managers/SdlManagerListener.java

```diff
@@ -62,8 +62,9 @@
 	 * is not supported.
 	 *
 	 * @param language The language of the connected head unit the manager is trying to update the configuration.
+	 * @param hmiLanguage The hmiLanguage of the connected head unit the manager is trying to update the configuration.
 	 * @return An object of LifecycleConfigurationUpdate if the head unit language is supported,
 	 * otherwise null to indicate that the language is not supported.
 	 */
-	LifecycleConfigurationUpdate managerShouldUpdateLifecycle(Language language);
+	LifecycleConfigurationUpdate managerShouldUpdateLifecycle(Language language, Language hmiLanguage);
```

Comment:
Per the proposal, The old method has to be deprecated.

Reply:
This has been fixed.